docs/rules/type-properties.md

```diff
@@ -44,6 +44,22 @@ type A = {
 }
 ```
 
+## Options
+
+```json
+{
+  "sort/type-properties": ["error", { "caseSensitive": false, "natural": true }]
+}
+```
+
+- `caseSensitive` - if `true`, enforce properties to be in case-sensitive order.
+  Default is `false`.
+- `natural` - if `true`, enforce properties to be in natural order. Default is
+  `true`. Natural Order compares strings containing combination of letters and
+  numbers in the way a human being would sort. It basically sorts numerically,
+  instead of sorting alphabetically. So the number 10 comes after the number 3
+  in Natural Sorting.
```

Comment:
```suggestion
  `true`. Natural order compares strings containing combination of letters and
  numbers in the way a human being would sort. It basically sorts numerically,
  instead of sorting alphabetically. So the number 10 comes after the number 3
  in natural sorting.
```

Reply:
Same for the other markdown files.

Reply:
Also, adding a note about the default value like you have for `caseSensitive` would be good.